explainaboard/analysis/analyses.py

```diff
@@ -391,11 +396,18 @@ def perform(
             if x not in cases[0].features:
                 raise RuntimeError(f"combo analysis: feature {x} not found.")
 
-        combo_map: dict[tuple[str, ...], int] = {}
+        combo_map: dict[tuple[str, ...], list] = {}
```

Comment:
1. As I commented, we don't need counts. This dict can hold only the list of IDs.
2. Defaultdict may help.

```suggestion
        combo_map: DefaultDict[tuple[str, ...], list[int]] = defaultdict(list)
```